WordPress/src/main/res/values/strings.xml

```diff
@@ -3134,7 +3134,7 @@
     <string name="quick_start_sites">Next Steps</string>
     <string name="quick_start_sites_type_customize">Customize your site</string>
     <string name="quick_start_sites_type_grow">Grow your audience</string>
-    <string name="quick_start_sites_type_get_to_know_app">Get to know the WordPress app</string>
+    <string name="quick_start_sites_type_get_to_know_app">Get to know the app</string>
```

Comment:
This string copy didn't get "frozen" and included in the strings to be translated during code freeze, since that change didn't land at the time the code freeze was made. This means that by default, this new copy would not be included in the translation pipeline and sent to GlotPress for the 19.9 cycle… unless we update the frozen strings as well.

Thankfully I spotted that just in time so I've updated the copy in the frozen strings via 375541878367a9e1fb64db007661e84105c17bdd ; but just a reminder for next time that whenever you update any string on a PR targeting the `release/*` branch (in other words, post-code freeze), please inform your release manager to check if the change in the string can still be included into the translation pipeline for this cycle. If the change is made later in the sprint and strings have already been sent to GlotPress, it might be too late to include any new translation post-code-freeze, but if the change happens soon enough in the sprint, we might be able to handle the exceptions and try to include the change so that it gets picked up in GlotPress for this cycle in time… as long as we're informed soon enough about it too 🙃

Reply:
Makes sense. 👍  We'll keep this in mind. 🙇‍♀️